README.md

```diff
@@ -1,43 +1,47 @@
 # go-clang-phoenix-gen [![GoDoc](https://godoc.org/github.com/zimmski/go-clang-phoenix-gen?status.png)](https://godoc.org/github.com/zimmski/go-clang-phoenix-gen) [![Build Status](https://travis-ci.org/zimmski/go-clang-phoenix-gen.svg?branch=master)](https://travis-ci.org/zimmski/go-clang-phoenix-gen) [![Coverage Status](https://coveralls.io/repos/zimmski/go-clang-phoenix-gen-gen/badge.png?branch=master)](https://coveralls.io/r/zimmski/go-clang-phoenix-gen?branch=master)
 
-Native Go bindings for the C API of Clang.
+Generate native Go bindings for Clang's C API.
 
-## Installation
+## I found a bug/missing a feature in go-clang
+
+Please go through the [open issues](/issues) in the tracker first. If you cannot find your request just open up a [new issue](/issues/new).
+
+## Where are the bindings?
+
+The Go bindings are placed in their own repositories to provide the correct bindings for the corresponding Clang version:
+
+- [v3.4](github.com/zimmski/go-clang-phoenix-v3.4)
+- [v3.6](github.com/zimmski/go-clang-phoenix-v3.6)
+- [v3.7](github.com/zimmski/go-clang-phoenix-v3.7)
+
+## Install go-clang-phoenix-gen
 
 ```bash
 CGO_CFLAGS="-I`llvm-config --includedir`" \
   CGO_LDFLAGS="-L`llvm-config --libdir`" \
-  go get github.com/zimmski/go-clang-phoenix-gen
+  go get github.com/zimmski/go-clang-phoenix-bootstrap github.com/zimmski/go-clang-phoenix-gen
 ```
 
-## Example
-
-An example on how to use the AST visior of Clang can be found in [/cmd/go-clang-dump/main.go](/cmd/go-clang-dump/main.go)
-
 ## How to develop for go-clang-phoenix-gen?
 
 You want to contribute to go-clang-phoenix-gen? GREAT! If you are here because of a bug you want to fix or a feature you want to add you can just read on, otherwise we have a list of [open issues in the tracker](/issues). Just choose something you think you can work on and discuss your plans in the issue by commenting on it.
 
-The development branch of go-clang-phoenix-gen is not `master` it is `bootstrap`. `master` just holds the C API bindings of Clang for the latest stable Clang version. We therefore only accept changes based on the `bootstrap` branch.
+This repository, the [gen](github.com/zimmski/go-clang-phoenix-gen) repository, holds the code to generate new bindings out of the headers of Clang's C API. They bindings are then bootstraped using the [bootstrap](github.com/zimmski/go-clang-phoenix-bootstrap) repository. The `bootstrap` repository holds basic files, like the CI configuration and a Makefile, as well as additional code to make the bindings more complete and powerful.
```

Comment:
This repository, [gen](github.com/zimmski/go-clang-phoenix-gen), holds the code to generate new bindings from headers of Clang's C API. These bindings are then bootstraped using the [bootstrap](github.com/zimmski/go-clang-phoenix-bootstrap) repository. The `bootstrap` repository holds all basic files, like CI configuration and a Makefile, as well as some additional code to make the bindings more complete and powerful.